setup.py

```diff
@@ -15,7 +16,7 @@ def main():
         all_deps += extras[group_name]
     extras['all'] = all_deps
 
-    base_libs = ['neptune-client>=0.5.4', 'lightgbm']
+    base_libs = ['neptune-client>=0.9.0', 'lightgbm', 'matplotlib', 'scikit-plot', 'graphviz']
```

Comment:
@shnela let's make `graphviz` and optional lib, not required dependency. This is because `graphviz` depends on some C libs that user need to install manually, procedure is different for Win/Linux/MacOS.

Check how it's done in XGBoost: https://github.com/neptune-ai/neptune-xgboost/blob/master/neptune_xgboost/impl/__init__.py#L54:L61

Graphsviz is not required: https://github.com/neptune-ai/neptune-xgboost/blob/master/setup.py#L17

Reply:
yep, You're right, let's keep it like you proposed.

Reply:
So what about Xgboost?

We have described problem in that library, which also requires `graphviz`, right?